source/core/owt_base/RawTransport.cpp

```diff
@@ -65,20 +82,44 @@ void RawTransport<prot>::close()
 template<Protocol prot>
 void RawTransport<prot>::createConnection(const std::string& ip, uint32_t port)
 {
+    if (std::fstream{kServerCrt}) {
+        m_ssl = true;
```

Comment:
TODO: in the future please make sure for known CA signed cert, you don't need to depend on this file to enable SSL.